modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/content/Language.properties

```diff
@@ -51,9 +53,12 @@ save-form=Save Form
 saving=Saving
 select-an-existing-form-or-add-a-form-to-be-displayed-in-this-application=Select an existing form or add a form to be displayed in this application.
 select-form=Select Form
+show-csv-warning=Show CSV Warning
+show-warning=Show warning
 someone=Someone
 sorry-something-wrong-happened=Sorry, something wrong happened
 stay=Stay
+the-csv-warning-info=Warning: This CSV file Contains user supplied inputs. Opening a CSV file may be dangerous.
```

Comment:
Change `the-csv-warning-info` to `csv-warning-message`
Change the value to: `Warning: This CSV file contains user supplied inputs. Opening a CSV file in a spreadsheet program may be dangerous.`